trezor-crypto/CMakeLists.txt

```diff
@@ -22,8 +22,7 @@ add_library(TrezorCrypto
     src/cash_addr.c
     src/memzero.c
     src/scrypt.c
-    src/ripple/base58.c
-)
+    src/ripple/base58.c)
```

Comment:
Why change this format?

Reply:
I was try out crc to class and cleaned extra line as on java :).